src/test/java/nextstep/cucumber/steps/LineStepDef.java

```diff
@@ -0,0 +1,67 @@
+package nextstep.cucumber.steps;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.cucumber.datatable.DataTable;
+import io.cucumber.java.en.Given;
+import io.cucumber.java8.En;
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import nextstep.cucumber.AcceptanceContext;
+import nextstep.subway.line.application.dto.LineResponse;
+import nextstep.subway.station.application.dto.StationResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class LineStepDef implements En {
+    @Autowired
+    private AcceptanceContext context;
+
+    @Given("지하철 노선들을 생성 요청하고")
+    public void 지하철_노선들_생성(DataTable table) {
+        List<Map<String, String>> maps = table.asMaps();
+        maps.stream()
+                .forEach(it -> {
+                    Map<String, String> params = new HashMap<>();
+                    params.put("name", it.get("name"));
+                    params.put("color", it.get("color"));
+                    params.put("upStationId", ((StationResponse) context.store.get(it.get("upStation"))).getId().toString());
+                    params.put("downStationId", ((StationResponse) context.store.get(it.get("downStation"))).getId().toString());
+                    params.put("distance", it.get("distance"));
+                    params.put("duration", it.get("duration"));
+                    ExtractableResponse<Response> response = RestAssured.given().log().all()
+                            .body(params)
+                            .contentType(MediaType.APPLICATION_JSON_VALUE)
+                            .when()
+                            .post("/lines")
+                            .then().log().all()
+                            .extract();
```

Comment:
`AcceptanceTestUtil`에 정의해두신 메서드를 활용해 볼 수 없을까요? 지금처럼 `RestAssured`를 통해 각각의 API를 호출하는 로직들을 메서드로 잘 정의해두신다면 편하게 테스트를 작성하실 수 있지 않을까 생각합니다.
```suggestion
                    ExtractableResponse<Response> response = AcceptanceTestUtil.노선_생성_Extract(params);
```